openbook_notifications/serializers.py

```diff
@@ -15,6 +15,9 @@ class ReadNotificationsSerializer(serializers.Serializer):
     max_id = serializers.IntegerField(
         required=False,
     )
+    types = serializers.CharField(
```

Comment:
Use a ListField with inner charfields which only accept the valid notification types, I believe this is a constant in the Notifications class.

This means that we first have to parse the querystring into a list of strings, and then pass it to this serializer

Reply:
Two questions:
1) How can I limit a CharField to only accept specific values?
2) Where would I do the query string parsing?

(Sorry if these are really easy things... I still only have a very vague grasp of how django is built up, and this is my first time using python. Can take a long time to figure even simple things out sometimes...)